kofta/public/locales/de/translation.json

```diff
@@ -172,13 +172,13 @@
 			"togglePushToTalkKeybind": "toggle push-to-talk keybind"
 		},
 		"wsKilled": {
-			"description": "Websocket was killed by the server. This usually happens when you open the website in another tab.",
-			"reconnect": "reconnect"
+			"description": "Websocket wurden vom Server gekillt. Dies kann passieren, wenn du die Webseite in einem neuen Tab erneut öffnest.",
+			"reconnect": "Neu verbinden."
 		},
 		"deviceNotSupported": {
-			"notSupported": "Your device is currently not supported. You can create an",
-			"linkText": "issue on GitHub",
-			"addSupport": "and I will try adding support for your device."
+			"notSupported": "Dein Gerät ist zurzeit leider nicht unterstützt. Du kannst ein",
```

Comment:
"einen [...] Issue" - but actually I would use another word for "issue", think of that a regular user might not know what an Issue is

Reply:
Ik, but how would you describe Issue differently? I would either drop it altogether or keep Issue. I mean because people who don't know what an issue is, won't open one.